security/i2pd/Portfile

```diff
@@ -0,0 +1,164 @@
+
+set i2pddir             ${prefix}/var/db/${name}
+
+post-patch {
+    reinplace "s|@PREFIX@|${prefix}|g" ${worksrcpath}/contrib/i2pd.conf
+    reinplace "s|@I2PDDIR@|${i2pddir}|g" ${worksrcpath}/contrib/i2pd.conf
+}
+
+set i2puser             _i2p
+add_users               ${i2puser} group=${i2puser} home=${i2pddir}
+
+cmake.source_dir        ${worksrcpath}/build
+
+# Minimum required is C++11, C++17 used if supported.
+compiler.cxx_standard   2011
+
+# https://github.com/PurpleI2P/i2pd/issues/1798
+compiler.blacklist-append \
+                        {clang < 600}
+
+# These are defaults, but better have them explicit:
+configure.args-append   -DWITH_LIBRARY=ON \
+                        -DWITH_BINARY=ON \
+                        -DWITH_STATIC=OFF
+
+# See Makefile.osx
+configure.ldflags-append "-lz -lcrypto -lpthread"
+configure.ldflags-append "-dead_strip -dead_strip_dylibs -bind_at_load"
+
+if {${build_arch} in [list ppc ppc64]} {
+    configure.args-append \
+                        -Dppc_support=TRUE \
+                        -DWITH_AESNI=OFF
+    # Temporary fix, enabling Boost atomics in place of default GCC.
+    # See: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=107590
+    # https://github.com/PurpleI2P/i2pd/issues/1726
+    # https://github.com/boostorg/smart_ptr/issues/105
+    configure.cxxflags-append \
+                        -DBOOST_SP_USE_STD_ATOMIC
+}
+
+post-destroot {
+    xinstall -d ${destroot}${prefix}/etc/${name}
+    foreach f {i2pd.conf subscriptions.txt tunnels.conf} {
+        if {[file exists ${worksrcpath}/contrib/${f}]} {
+            xinstall -m 0664 ${worksrcpath}/contrib/${f} ${destroot}${prefix}/etc/${name}
+        }
+    }
+    if {[file exists ${worksrcpath}/contrib/tunnels.d]} {
+        copy ${worksrcpath}/contrib/tunnels.d ${destroot}${prefix}/etc/${name}
+    }
+    if {[file exists ${worksrcpath}/contrib/certificates]} {
+        xinstall -d ${destroot}${prefix}/share/${name}
+        copy ${worksrcpath}/contrib/certificates ${destroot}${prefix}/share/${name}
+    }
+    xinstall -d ${destroot}${prefix}/share/doc/${name}
+    foreach f {ChangeLog LICENSE Makefile.osx README.md} {
+        if {[file exists ${worksrcpath}/${f}]} {
+            xinstall -m 0644 ${worksrcpath}/${f} ${destroot}${prefix}/share/doc/${name}
+        }
+    }
+    # backup conf files
+    foreach f {i2pd.conf subscriptions.txt tunnels.conf tunnels.d} {
+        if {[file exists ${prefix}/etc/${name}/${f}]} {
+            move ${destroot}${prefix}/etc/${name}/${f} ${destroot}${prefix}/etc/${name}/${f}.new
+            copy ${prefix}/etc/${name}/${f} ${destroot}${prefix}/etc/${name}/${f}.mp_backup
+            file attributes ${destroot}${prefix}/etc/${name}/${f}.mp_backup -owner ${i2puser} -group ${i2puser} -permissions 0664
+        }
+    }
+}
+
+post-install {
+    ui_msg "******************************************************"
+    ui_msg "*"
+    ui_msg "* i2pd status can be checked on: https://127.0.0.1:7070"
```

Comment:
```suggestion
    ui_msg "* i2pd status can be checked on: http://127.0.0.1:7070"
```

Reply:
@catap Gimme a sec, I will update this and a correction above.

Could you please take a look how to enable tests properly?